.changes/unreleased/NOTES-20230213-155943.yaml

```diff
@@ -0,0 +1,8 @@
+kind: NOTES
+body: 'helper/resource: ImportStateVerify will now return any timeouts block differences
+  which are significant on Terraform 1.3.8 or later. If ImportStateVerify reports
+  any differences, ensure any terraform-plugin-sdk/v2 dependency is version 2.25.0
+  or later.'
```

Comment:
We can nix this changelog note if/when terraform-plugin-sdk is released with https://github.com/hashicorp/terraform-plugin-sdk/pull/1146 and this repository updates its dependency.